components/Common/Select/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { randomBytes } from 'crypto';
+
+import { ChevronDownIcon } from '@heroicons/react/24/outline';
+import * as Primitive from '@radix-ui/react-select';
+import { useId, useMemo } from 'react';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+type SelectProps = {
+  values: string[];
+  defaultValue?: string;
+  placeholder?: string;
+  dropdownLabel?: string;
+  label?: string;
+  onChange?: (value: string) => void;
+};
+
+const Select: FC<SelectProps> = ({
+  values = [],
+  defaultValue,
+  placeholder,
+  label,
+  dropdownLabel,
+  onChange,
+}) => {
+  const id = useId();
+  const componentKey = useMemo(() => randomBytes(16).toString('hex'), []);
```

Comment:
Again, I believe that since a Select/List requires values to be unique, I do believe that using the value as a key is enough.

I think that having a key for uniqueness per component is unnecessary here, as React doesn't compute these things in relation of other components. (My suggestion is to remove this) and just use the value and nothing more as a key